gradle/libs.versions.toml

```diff
@@ -130,6 +131,8 @@ rxjava3-core = { module = "io.reactivex.rxjava3:rxjava", version.ref = "rxjava3"
 rxjava3-kotlin = { module = "io.reactivex.rxjava3:rxkotlin", version.ref = "rxkotlin3" }
 sparkbutton = { module = "com.github.connyduck:sparkbutton", version.ref = "sparkbutton" }
 unified-push = { module = "com.github.UnifiedPush:android-connector", version.ref = "unified-push" }
+exoplayer-core = { module = "com.google.android.exoplayer:exoplayer-core", version.ref = "exoplayer" }
+exoplayer-ui = { module = "com.google.android.exoplayer:exoplayer-ui", version.ref = "exoplayer" }
```

Comment:
I don't understand why we need the changes in this file when we already include this information in build.gradle. Also, this seems to be including exoplayer whereas build.gradle is including media3. That is almost certainly wrong. However without these lines the fails with a [url=https://matrix.to/#/!PWkUGGpeMZWDPmdfRE:matrix.org/$1676600712263985pxBEA:matrix.org?via=m.charlag.space&via=matrix.org&via=tchncs.de]mysterious nullPointerException[/url]